scripts/analyze_failed_jobs/action_plugins/parsejobs.py

```diff
@@ -0,0 +1,40 @@
+import glob, pickle, buildbot, re, sys, cStringIO as StringIO
```

Comment:
Yes, please add a `README.md` as you describe.

However, Tom is right, please add a short docstring. 

```
"""
Ansible module for finding failed buildbot jobs.

Stores results in ``out/count.csv``, which has columns for $FOO, $BAR, $BAZ.
"""
```